swatch-contracts/ct/java/tests/ContractsComponentTest.java

```diff
@@ -21,43 +21,20 @@
 package tests;
 
 import static domain.Contract.buildRosaContract;
-import static java.util.concurrent.TimeUnit.MINUTES;
-import static java.util.concurrent.TimeUnit.SECONDS;
-import static org.awaitility.Awaitility.await;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.greaterThan;
-import static org.hamcrest.Matchers.is;
-import static org.hamcrest.Matchers.lessThan;
-import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import com.redhat.swatch.component.tests.utils.RandomUtils;
 import com.redhat.swatch.configuration.util.MetricIdUtils;
-import com.redhat.swatch.contract.test.model.SkuCapacityV2;
-import com.redhat.swatch.contract.test.model.SubscriptionEventType;
 import domain.BillingProvider;
 import domain.Contract;
-import domain.Offering;
-import domain.Product;
-import domain.Subscription;
-import io.restassured.response.Response;
-import java.time.OffsetDateTime;
 import java.util.Map;
-import java.util.Optional;
-import org.apache.http.HttpStatus;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
+@Tag("contract")
```

Comment:
Can this tag be removed after [this discussion](https://github.com/RedHatInsights/rhsm-subscriptions/pull/5383#discussion_r2610309201)?